moveit_ros/planning/planning_pipeline/include/moveit/planning_pipeline/planning_pipeline.h

```diff
@@ -163,6 +162,14 @@ class MOVEIT_PLANNING_PIPELINE_EXPORT PlanningPipeline
   {
     return false;
   }
+  [[deprecated("Please use getPlannerPluginNames().")]] const std::string& getPlannerPluginName() const
+  {
+    return pipeline_parameters_.planning_plugins.at(0);
+  }
+  [[deprecated("Removed from API.")]] const planning_interface::PlannerManagerPtr& getPlannerManager()
```

Comment:
Is there no further rationale for this being removed from the API that we can display?

Reply:
I am hesitant if raw access to the planner itself should be provided by the pipeline class :thinking: Looks like the main reason for that has been so you can read configs from the planner but that could be done more explicitly